ddl/column.go

```diff
@@ -1076,6 +1076,9 @@ func (w *worker) doModifyColumnTypeWithData(
 		changingColumnUniqueName := changingCol.Name
 		changingCol.Name = colName
 		changingCol.ChangeStateInfo = nil
+		if err = changingCol.SetOriginDefaultValue(nil); err != nil {
```

Comment:
great! should comment here? original default value is no longer used after CTC because all of it have been casted to real row.